### PR TITLE
Awesome book2

### PR DESCRIPTION
**Pull request by ademibowale**
- [ ] This is a pair program project 
- [ ] I refactor the project to add three new sections on top of the two sections we have already completed
- [ ] Added a new javascript  features t .
- [ ] My  code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,207 @@
+* {
```
Comment (vechicin) [inline styles.css:1]: Please change this file's name to `index.css`.


Screen Shot 2022-07-04 at 10 55 02

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @ademibowale ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS> 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (ademibowale) [reply styles.css]: thanks

**Comment by ademibowale** [comment]
- [ ] This is a pair program project 
- [ ] I refactor the project to add three new sections on top of the two sections we have already completed
- [ ] Added a new javascript  features t .
- [ ] My  code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

**Review by keroloussamy — APPROVED** [review]
Hi @ademibowale, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**